### DISCUSSION TITLE
Request failed with status code 404

### DISCUSSION THREAD

**Discussion by jdarias** [jesec/flood#227] [Q&A] [answered]
I´m trying to configure flood to be accesible from my local network and so far i´ve followed the tutorial in the wiki to configure nginx as a nested route as instructed [in here](https://github.com/jesec/flood/wiki/Using-Flood-behind-a-web-server-(Nginx)#serve-from-a-nested-route). However, when I input my login details flood throws a **"Request failed with status code 404"** message.
I was able to access flood in the machine via 127.0.0.1:3000 and i set my details, the tcp config for SCGI and it worked, I was able to see my downloads and add a rss feed. 
However, when trying the nginx config from the wiki the aforementioned message appears as i enter my credentials. How can I fix it?
I´m attaching my configs
This is my flood@user.service in systemd
```[Unit]
Description=Flood Service for %I
After=network.target

[Service]
User=%I
Group=%I
Type=simple
KillMode=process
ExecStart=/usr/bin/env flood --host:"0.0.0.0" --rthost:"127.0.0.1" --rtport:5000 --baseuri:"/flood"
Restart=on-failure
RestartSec=3
StandardOutput=syslog
StandardError=syslog
SyslogIdentifier=notell

[Install]
WantedBy=multi-user.target
```

This is the configuration for nginx:

```       
        include /etc/nginx/conf.d/*.conf;
        include /etc/nginx/sites-enabled/*;
        server {
            listen 0.0.0.0:8008;
            server_name ngnix-rtorrent;
            access_log /var/log/nginx/rtorrent.access_log;
            error_log /var/log/nginx/rtorrent.error_log;
            location /RPC2 {
                scgi_pass   127.0.0.1:5000;
                include     scgi_params;
            }

        }

        server {
                server_name raspberrypi.ae.local;

                location /flood/api {
                  proxy_pass http://127.0.0.1:3000;
                }

                location /flood/ {
                  alias /usr/lib/node_modules/flood/dist/assets/;
                  try_files $uri /flood/index.html;
                }
        }
}
```

I have configured SCGI in rtorrent like this in rtorrent.rc
```
network.scgi.open_port = "127.0.0.1:5000"
```

PS: even though I start flood with --host:"0.0.0.0" i cannot access it remotely from the machine name or its ip address.

**Comment by jesec** [comment, marked as answer]
Use `=` instead of `:`.

**Reply by jesec** [reply]
This is irrelevant to Flood. I noticed that you have exposed rTorrent RPC interface with `/RPC2` route to 0.0.0.0 aka anywhere, without authentication apparently. 

Seriously don’t do that. It is extremely dangerous. See `Security Precautions` section of wiki and read https://www.f5.com/labs/articles/threat-intelligence/rtorrent-client-exploited-in-the-wild-to-deploy-monero-crypto-miner. 

Also, it is recommended to use socket file mode instead of port mode.

**Reply by jdarias** [reply]
Thank you so much! indeed I was doubtful about using = or : in the systemd script.
Regarding the security issue you mention, could you recommend a solution for my case? I´m not interested in accessing rtorrent nor flood from outside my network, but I want it to be accesible from all hosts within it.
Again Thanks, it´s great to have this app updated, I installed it from the old repo when it was 1.0 and that was painful. It's nice to have the process simplified and the app bug free and finally mobile friendly

**Reply by jesec** [reply]
Unless you have an integration that interacts with rTorrent RPC interface directly, and there is no alternative, you do not need to expose the rTorrent RPC interface. It is not recommended for an application to interact with rTorrent RPC interface remotely.
